### PR TITLE
2 digit routing number search

### PR DESCRIPTION
**Pull request by bkmoovio**
#15

### CODE REVIEW

File: fileErrors.go [hunk 1]
```diff
@@ -12,7 +12,13 @@ import (
 )
 
 // ErrFileTooLong is the error given when a file exceeds the maximum possible length
-var ErrFileTooLong = errors.New("file exceeds maximum possible number of lines")
+var (
+	ErrFileTooLong = errors.New("file exceeds maximum possible number of lines")
+	// Similar to FEDACH site
+	ErrToManyRecords = errors.New("there are more than 499 FedACH Participants matching your search criteria. " +
```
Comment (adamdecaf) [inline fileErrors.go]: Should this be Printf formatted with `MaximumRecordsReturned` and other values?

File: validators.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+var (
+	numericRegex = regexp.MustCompile(`[^0-9]`)
+	msgNumeric   = "is not 0-9"
+)
+
+// validator is common validation and formatting of golang types to fed type strings
+type validator struct{}
+
+// isNumeric checks if a string only contains ASCII numeric (0-9) characters
+func (v *validator) isNumeric(s string) error {
```
Comment (adamdecaf) [inline validators.go]: A regex might be slow compared to checking each rune:

```Go
for i := range s {
	if s[i] < '0' || s[i] > '9' {
		return errors.New(msgNumeric)
	}
}
```

File: fileErrors.go (hunk 1 above)
Comment (bkmoovio) [reply fileErrors.go]: Sure

File: validators.go (hunk 2 above)
Comment (bkmoovio) [reply validators.go]: I believe I tested that many moons ago and it was negligible

File: fileErrors.go (hunk 1 above)
Comment (bkmoovio) [reply fileErrors.go]: I will remove the hard-coding of 499.  Knowing that actual value returned probably not needed.

Comment (bkmoovio) [reply fileErrors.go]: ehh, I'll add it in anyway

**Pull request merged**